drivers/sensor/icp10125/icp10125.c

```diff
@@ -194,29 +190,26 @@ static int icp10125_measure(const struct i2c_dt_spec *i2c, const struct icp10125
 
 	rc = i2c_write_dt(i2c, (uint8_t *)&cmds[mode], sizeof(cmds[mode]));
 	if (rc < 0) {
-		LOG_ERR("Failed to start measurement.\n");
+		LOG_ERR("Failed to start measurement.");
 		return rc;
 	}
 
 	/* Wait for the sensor to become readable.
 	 * First wait for the typical time and then read.
 	 * If that fails, wait until the time to surely became readable.
 	 */
-	k_sleep(K_USEC(conv_time_typ[mode]));
-	if (i2c_read_dt(i2c, (uint8_t *)sensor_data, sizeof(sensor_data[0]) * data_num) < 0) {
-		k_sleep(K_USEC(conv_time_max[mode] - conv_time_typ[mode]));
-		rc = i2c_read_dt(i2c, (uint8_t *)sensor_data, sizeof(sensor_data[0]) * data_num);
-		if (rc < 0) {
-			LOG_ERR("Failed to read measurement.\n");
-			return rc;
-		}
+	k_sleep(K_USEC(conv_time_max[mode]));
```

Comment:
I think the comment for this code needs updating, since you're not anymore waiting for typical time.

On a related topic, why have you made this change?

Reply:
because we don't care about perfect time optimisation here and the logic before resulted in i2c errors printed (which we can't/should not suppress) very very often as from testing the typical time was often not enough for a successful read. The chip has a very rubbish read mechanism!
With this approach we ensure there are no unnecessary error logs, if there is an error after max time it's a valid error

Reply:
OK makes sense, can you update the comment in the file, please? I'll then resolve the conversation.

Reply:
done